README.md

```diff
@@ -1 +1,37 @@
-# angular-what-is-a-directive-readme
\ No newline at end of file
+# What is a Directive?
+
+## Overview
+
+You might not have noticed, but we've used directives already in our previous labs. Directives are essential to learning Angular, so hold in tight!
```

Comment:
really nice context - we've been using directives the whole time, we just didn't notice